healthcheck/versionFile.js

```diff
@@ -19,7 +19,7 @@ const versionFile = () => {
 
 const version = () => {
   return versionFile().then(props => {
-    return (props.version) ? (props.build) ?  props.version + "-" + props.build : props.version : defaultObj.version
+    return (props.version) ? (props.number) ?  props.version + "-" + props.number : props.version : defaultObj.version
```

Comment:
can you remind me where this is used?
i.e. I can't see a build or a number here:
https://www-dev.moneyclaim.reform.hmcts.net/health
this has a number (but from a different library):
https://www-dev.moneyclaim.reform.hmcts.net/info